modules/t/fastaSequenceAdaptor.t

```diff
@@ -25,11 +25,19 @@ use Bio::EnsEMBL::Test::MultiTestDB;
 
 use File::Spec;
 use File::Temp qw/:seekable tempdir/;
-use Bio::EnsEMBL::Utils::IO::FASTASerializer;
 use Bio::EnsEMBL::DBSQL::FastaSequenceAdaptor;
-use Bio::EnsEMBL::Utils::IO::FileFaidx;
 use Bio::EnsEMBL::Utils::IO qw/work_with_file/;
 
+# Check if ensembl-io repo is available
+eval {
+  require Bio::EnsEMBL::Utils::IO::FASTASerializer;
+  require Bio::EnsEMBL::Utils::IO::FileFaidx;
+};
+if ($@) {
```

Comment:
This might work as expected in case of _require_ but as a rule, the reliable way of telling whether an eval has failed is to look at its return value and only use _$@_ to propagate the error message, if any.

Reply:
I used the 'standard' ensembl way (slice.t, schemaPatches.t, housecleaning_perlCritic.t and circularSlice.t all use it)